src/roiextractors/extractors/suite2p/suite2psegmentationextractor.py

```diff
@@ -70,7 +97,11 @@ def __init__(
         self._image_mean = self._summary_image_read("meanImg")
 
     def _load_npy(self, filename, mmap_mode=None):
-        file_path = self.folder_path / f"plane{self.plane_no}" / filename
+        if self._search_plane_subdirectory:
+            file_path = self.folder_path / f"plane{self.plane_no}" / filename
+        else:
+            file_path = self.folder_path / filename
+        assert file_path.exists(), f"File {file_path} does not exist, but is required for this extractor."
         return np.load(file_path, mmap_mode=mmap_mode, allow_pickle=mmap_mode is None)
```

Comment:
```suggestion
        file_path = self.folder_path / f"plane{self.plane_no}" / filename
        if not file_path.exists():
            return
        return np.load(file_path, mmap_mode=mmap_mode, allow_pickle=mmap_mode is None)
```

Reply:
If accepted, this would allow you to pretty much revert all the changes in the `__init__` that used the locally scoped function there (including all the new arguments)

Reply:
lgtm